docs/source/env_config.rst

```diff
@@ -0,0 +1,22 @@
+* **DEMO** A non-empty value causes nivio to start in demo mode with prepared data.
+* **GITHUB_JWT** To connect via JWT token as a GitHub App:
+* **GITHUB_LOGIN** GitHub user name (can also be used to connect as organisation with oauth
+* **GITHUB_OAUTH** To connect to GitHub via Personal access token
+* **GITHUB_PASSWORD** GitHub password (for username/pw login)
+* **GITLAB_HOST_URL** The full URL to the GitLab API, e.g. http://your.gitlab.server.com
+* **GITLAB_PASSWORD** GitLab OAuth login password (optional)
+* **GITLAB_PERSONAL_ACCESS_TOKEN** the personal token to access the GitLab API (optional)
+* **GITLAB_USERNAME** GitLab OAuth login username (optional, if used password is also required)
+* **KUBERNETES_MASTER** K8s master url (optional), all variables from https://github.com/fabric8io/kubernetes-client#configuring-the-client can be used
```

Comment:
I still think that this suggests that all k8s variables are supported right now. As long as this does not really go into the release and the k8s support will also be delivered with the next release, this is fine ;)

Reply:
We talked about this directly, k8s support is alpha, but the library we use already supports the mentioned environment variables.